requirements.txt

```diff
@@ -1 +0,0 @@
-enum34==1.1.6
```

Comment:
This file is now empty. Could you please remove it altogether and make changes in setup.py accordingly?

Reply:
Fixed